### DISCUSSION TITLE
Introduce threads.net

### DISCUSSION THREAD

**Discussion by lippoliv** [jpanther/congo#835] [Ideas]
Thanks @jpanther for your effort with Congo :) I'm new to Hugo, I saw your theme, I liked it, I started a new project using it. Works like a charm right now.

As time goes by, I introduce "new" stuff to my project. Recently I "connected" several social media platforms. I didn't found out how to linke Instagrams new Service "threads" in the config? I think currently it's not supported.

Would you add it? I hope it's not too much effort...

I know it's not too big right now (at least in Germany) but it will grow.

**Comment by jpanther** [comment]
The Threads icon is available in the theme. Are you talking about making a link to your profile or are you referring to sharing links?

**Reply by lippoliv** [reply]
Yes, I think about this two places: 
1. https://github.com/jpanther/congo/blob/dev/data/sharing.json
2. `params.author.links` (https://github.com/jpanther/congo/blob/dev/config/_default/languages.en.toml#L20-L64)

If it's there already, how to use it?

**Reply by jpanther** [reply]
In author links simply use `{ threads = "url" }` and it should work. I'll update the config comments so it's more obvious but generally you can use any theme icon in the author links. It's not currently available in the article sharing but I'll look to add this in a future update.
